code/modules/paperwork/faxmachine.dm

```diff
@@ -1,6 +1,7 @@
 var/list/obj/machinery/photocopier/faxmachine/allfaxes = list()
 var/list/admin_departments = list("Central Command", "Sol Government")
 var/list/alldepartments = list()
+var/broadcast_departments = "Stationwide broadcast (WARNING)"
```

Comment:
Thanks for the heads-up. Makes sense to make these vars static class members. Amended in fb57de624b267af4bf3366a7d38409cc604060c2.